artemis-jdbc-store/src/main/java/org/apache/activemq/artemis/jdbc/store/sql/PropertySQLProvider.java

```diff
@@ -362,8 +363,12 @@ public Factory(SQLDialect dialect) {
          }
       }
 
-      public Factory(DataSource dataSource) {
```

Comment:
Can't we keep this method and call this(new JDBCConnectionProvider(ds));

Reply:
@uomik this is still missing in your latest changes

Reply:
@ehsavoie Fixed now. For some reason the commit did not make it to my last push.